ocppj/server.go

```diff
@@ -41,7 +43,7 @@ type InvalidMessageHook func(client ws.Channel, err *ocpp.Error, rawJson string,
 //	s := ocppj.NewServer(ws.NewServer(), nil, nil)
 //
 // The dispatcher's associated ClientState will be set during initialization.
-func NewServer(wsServer ws.Server, dispatcher ServerDispatcher, stateHandler ServerState, profiles ...*ocpp.Profile) *Server {
+func NewServer(wsServer ws.Server, dispatcher ServerDispatcher, stateHandler ServerState, msgHooks MessageHooks, profiles ...*ocpp.Profile) *Server {
```

Comment:
This is a breaking change and several custom implementations may be affected. I believe the added `SetMessageHooks` setter is good enough, so we don't have to also include it as a parameter in the constructor